hack/dind-cluster.sh

```diff
@@ -221,7 +221,7 @@ function wait-for-cluster() {
   oc="$(os::build::find-binary oc)"
 
   # wait for healthz to report ok before trying to get nodes
-  os::util::wait-for-condition "ok" "${oc} get --config=\"${kubeconfig}\" --raw=/healthz" "120"
```

Comment:
The function in question claims it uses `eval` but in fact does this:

```sh
  while ! $(${condition}); do
```

This means that yes, these quotes were incorrect and the actual value of `--config` that is passed to `oc get` is `"${kubeconfig}"`, literal quotes and all. @marun this type of thing is why I feel so strongly about _not_ having the `provision_util.sh` file, since we spent a lot of time and effort getting it right the first time in `os::cmd`.